src/components/Reports/Subscribers/SubscribersRedirect.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { InjectAppServices } from '../../../services/pure-di';
+import { useParams, Redirect } from 'react-router-dom';
+import SafeRedirect from '../../SafeRedirect';
+import { replaceSpaceWithSigns, extractParameter } from '../../../utils';
+import queryString from 'query-string';
+
+const SubscribersRedirect = ({ location }) => {
+  const { oldParam } = useParams();
+  const email = replaceSpaceWithSigns(extractParameter(location, queryString.parse, 'email'));
+  const section =
+    oldParam === 'subscriber-history' ? 'history' : oldParam === 'subscriber-gdpr' ? 'gdpr' : null;
+
+  if (!section) {
```

Comment:
What do you think about also check the email?

```suggestion
  if (!section || !email) {
```